samples/openthread/coprocessor/snippets/usb/usb.overlay

```diff
@@ -6,12 +6,12 @@
 
 / {
 	chosen {
-		zephyr,shell-uart = &cdc_acm_uart;
+		zephyr,ot-uart = &snippet_cdc_acm_uart;
```

Comment:
There is either a mixup in file names in here or something else is wrong. 

`samples/openthread/cli/usb.overlay → ...read/coprocessor/snippets/usb/usb.overlay`

`samples/openthread/coprocessor/usb.overlay → ...s/openthread/cli/snippets/usb/usb.overlay`

Reply:
actually, I think github mixed paths for these files. if you check them:
![image](https://github.com/nrfconnect/sdk-nrf/assets/115064000/a25fb4c3-290f-418f-ae2c-3895dd15b314)
![image](https://github.com/nrfconnect/sdk-nrf/assets/115064000/f9461f8e-c953-4846-beb4-16040a8ae012)
so it seems like gihtub's diff tool mixes coprocessor's usb overlay with cli's

Reply:
Ok let's wait for CI results.

Reply:
CI is green. Seems like Github's issue